lib/assessment.md

```diff
@@ -137,9 +150,9 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+It would look like below:
+[divTwo][divThree][divFive][divOne]
+[divFour]
```

Comment:
Close. Even though div Four clears a left float, divs Four and Five are still floated, so div Five will sit immediately to the right of div Four.